pkg/controller/command/verifiable/command.go

```diff
@@ -332,3 +411,67 @@ func (o *Command) generatePresentation(rw io.Writer, vc *verifiable.Credential,
 
 	return nil
 }
+
+// SignPresentation sign the verifiable presentation
+func (o *Command) SignPresentation(didDoc *did.Doc, vc *verifiable.Credential) (*verifiable.Presentation, error) { // nolint:lll
+	var s signer
+
+	pk, err := getPublicKeyID(didDoc)
```

Comment:
This should be part of request, this function is temporary work around in edge-services to decide which key to use. We shouldn't use that logic in aries. we should get it as an option in request. Some new param in request called 'proof options'

Reply:
what should proof options look like ??

Reply:
request.proofOption.{
vm,
signType,
}


}

Reply:
added